src/main/java/com/specialwarriors/conal/common/config/RedisConfig.java

```diff
@@ -0,0 +1,37 @@
+package com.specialwarriors.conal.common.config;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.redis.connection.ReactiveRedisConnectionFactory;
+import org.springframework.data.redis.connection.lettuce.LettuceConnectionFactory;
+import org.springframework.data.redis.core.ReactiveRedisTemplate;
+import org.springframework.data.redis.serializer.RedisSerializationContext;
+import org.springframework.data.redis.serializer.StringRedisSerializer;
+
+@Configuration
+public class RedisConfig {
+
+    @Value("${spring.data.redis.host}")
+    private String redisHost;
+    @Value("${spring.data.redis.port}")
+    private int redisPort;
```

Comment:
```suggestion
    @Value("${spring.data.redis.host}")
    private String redisHost;
    
    @Value("${spring.data.redis.port}")
    private int redisPort;
```
필드에 어노테이션을 사용할 경우 빈 라인을 두는 것이 가독성이 낫다고 생각하는데 검토하시고 의견 부탁드립니다.

Reply:
넵 검토 후 수정하겠습니다!